docs/guides/user/search-query-guide.md

```diff
@@ -1,48 +1,48 @@
 # Search within timeline
+
 ## Search queries
 
 Timesketch allows full text search within timelines. Good way to get started is by selecting one of pre-set search templates and adjusting them to the data in your timeline.
 
-Simple search queries relies on[ Query String Query](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-query-string-query.html) mini-language, but it is also possible to use the full potential of Elasticsearch query language in Advanced queries.
+Simple search queries relies on[ Query String Query](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-query-string-query.html) mini-language, but it is also possible to use the full potential of OpenSearch query language in Advanced queries.
```

Comment:
Instead of https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-query-string-query.html 
https://opensearch.org/docs/latest/opensearch/query-dsl/index/

Reply:
No, that is the docs for the DSL (json format) not the query string format. They have not ported over the query-string docs yet.

Reply:
ah okay, then we keep that link for now